.travis.yml

```diff
@@ -6,8 +6,9 @@ os:
   - linux
   - osx
 branches:
-  except:
-  - fleming
+  only:
```

Comment:
Sorry man; I told you that but actually speaking with Viv, can we just not specify anything (if CI speeds are decent)?